Automattic/jetpack
# Paid blocks: Prevent upgrade nudge from being rendered on the block list view

Fixes https://github.com/Automattic/jetpack/issues/26232

#### Changes proposed in this Pull Request:

Fixes a regression introduced by https://github.com/Automattic/jetpack/pull/25812 that caused the upgrade nudges of paid blocks to be displayed in the block list view.

To avoid that situation, we now use a more targeted CSS selector to correctly identify the element where the upgrade nudge should be added.

Before | After
--- | ---
![before](https://user-images.githubusercontent.com/1233880/190422589-ae64a2bd-0706-41c3-80ef-151341236c87.gif) |  ![after](https://user-images.githubusercontent.com/1233880/190422631-3035a4e4-1232-4dc0-bbbb-874c2a75c256.gif)

#### Other information:


#### Jetpack product discussion
N/A.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
- Apply the diff created Fusion to your WP.com sandbox
- Sandbox a WP.com simple site with a Free plan
- Create a post
- Insert a Columns block
- Add a paid block such as the Payment buttons block inside the Columns block
- Open the list view
- Move the paid block out of the Columns block
- Make sure the upgrade nudge is displayed above the block
